recipes/fftw/all/conanfile.py

```diff
@@ -19,13 +19,15 @@ class FFTWConan(ConanFile):
                "precision": ["double", "single", "longdouble"],
                "openmp": [True, False],
                "threads": [True, False],
-               "combinedthreads": [True, False]}
+               "combinedthreads": [True, False],
+               "SIMD": ["SSE", "SSE2", "AVX", "AVX2", "auto"]}
     default_options = {"shared": False,
                        "fPIC": True,
                        "precision": "double",
                        "openmp": False,
                        "threads": False,
-                       "combinedthreads": False}
+                       "combinedthreads": False,
+                       "SIMD": "auto"}
```

Comment:
```suggestion
                       "SIMD": "auto"}
```

No auto as default value of such option please. It may build binaries that consumers can't use on their arch.

Reply:
"auto" does nothing. I did it on purpose to exactly not break existing compilations. `auto` in fact does nothing. Do you have a better suggestion for a name?

Reply:
I went with your suggestion of using False instead.

Reply:
Oh ! I forgot: this patch should also be propagated to 3.3.8

Reply:
@SpaceIm changes have been pushed to 3.3.8 as well.